packages/server/src/db/db.service.ts

```diff
@@ -196,27 +201,40 @@ export default class DBService {
   }
 
   async getShelvesByUserId(userId: number) {
-    const userToShelves = await this.userToShelfRepository.find({ where: { userId }, relations: ['shelf'] });
+    const userToShelves = await this.userToShelfRepository.find({
+      where: { userId },
+      relations: ['shelf'],
+    });
     return userToShelves.map(u2s => u2s.shelf);
   }
 
   saveShelf(
     name: string,
     location: string,
     description: string,
-    pictureUrl: string,
+    pictureUrl: string
   ) {
-    return this.shelfRepository.save({ name, location, description, pictureUrl });
+    return this.shelfRepository.save({
+      name,
+      location,
+      description,
+      pictureUrl,
+    });
   }
 
   updateShelf(
     id: number,
     name: string,
     location: string,
     description: string,
-    pictureUrl: string,
+    pictureUrl: string
   ) {
-    return this.shelfRepository.update(id, { name, location, description, pictureUrl });
+    return this.shelfRepository.update(id, {
+      name,
+      location,
+      description,
+      pictureUrl,
```

Comment:
I would suggest removing coma after the last property for consistency in whole PR.

Reply:
Well, haven't we agreed on the trailing comma es-rule being set to `true` for objects?

Reply:
we've agreed that we allow to use trailing comma but it is optional

Reply:
So... May I leave it?